### ISSUE TITLE
Redesign of the main README

### ISSUE THREAD

**Issue by Aracki** [tldr-pages/tldr#3201]
## Suggestions

1. **_Header_** - Add a new square icon aligned to the right of the header. 
PS. (discussion about the new logo - #2986).

2. **_What is tldr?_** - I find it very useful, but I would change this huuuge screenshot with a smaller gif. (it can be done easily with tools like [terminalizer](https://github.com/faressoft/terminalizer))

3. **_Clients_** - Maybe we should discuss if we remove this one and leave only the link to our [wiki page](https://github.com/tldr-pages/tldr/wiki/TLDR-clients). AFAIK all the clients are more or less the same. I assume that the majority uses either `npm` (because it is mentioned in the header) or `c++` client (because the majority of MacOS users will automatically try to install CLI tool with `brew`). Another proposal would be to leave only the few which are most popular.

4. **_Similar projects_** - I would suggest that we remove this section. They are offering the same thing, but less mature than tldr-pages. So, I don't see the point of keeping it.

## Summary

I think the README is _too long_ and there are things _not so important_. It has to be more concise with a better design!

TLDR definitely has the potential to be used by many more people and to be a _**must-have tool within a developer terminal**_. I believe that it is **_the first few seconds_** that will make newcomers to install the tool and start using it.

**Comment by mebeim** [comment]
1. It could be done with a small image (e.g. 64x64). In such case though, the fact that the small logo already contains text doesn't make much sense. There should be a logo without text if this is done.

2. I don't really think a GIF would be of any help. A static image serves the purpose better: it's just a terminal screenshot showing a TLDR page, we don't need an animation. 

3. The list is indeed long... linking to a wiki page where everything is better organized would be ok IMHO. Maybe we could still suggest clients that are available through common package managers in the readme and leave a link to the wiki for the rest? 

4. I don't see any issue in having such section. I would keep it, and also not all of those are offering the same thing as TLDR.

---

> TLDR definitely has the potential to be used by many more people and to be a must-have tool within a developer terminal. I believe that it is the first few seconds that will make newcomers to install the tool and start using it.

This might be true, but we're not trying to "sell" anything here. People can decide for themselves if they want to use TLDR or not, and I think that for this purpose the first paragraph along with the currently displayed screenshot makes it easy enough: if interested, then you keep scrolling. The readme should give a complete enough overview, shorter doesn't necessarily mean better.

**Comment by sbrl** [comment]
### 1
If we do use the logo only, we shouldn't forget the alt text for accessibility purposes.

### 2
I'd probably agree with @mebeim here, but there's no reason why we can't shrink the native size of said image down a bit to make it appear smaller.

### 3
Indeed, a wiki page would probably be the best solution here.

### 4
I don't see anything wrong with the section. It might be worth revising it to mark inactive projects as such, but I think it's important to get an idea as to where tldr-pages comes from, and what it's similar to. It helps to highlight what tldr-pages _is_ about I think.

**Comment by andrik** [comment]
Hey guys,

I fully agree with @sbrl

**Comment by zdroid** [comment]
I suggest opening a new issue for **1** and **2**, or keeping this issue for those two points. Both require minimal effort, as we already have the logo and required graphics.

**3** is superseded by #4044.

I completely agree with you on **4**. None of those projects references our project, so there's no reason to give them such favor. They aren't as mature and don't have the same scope. A better idea may be to change the section's purpose from "Similar projects" to "Comparision to similar projects".

**Comment by sbrl** [comment]
Thanks for the update on this issue, @zdroid!

Those other projects might not have the same scope as tldr-pages, but I think that it's still important to reference them. A comparison sounds like an interesting idea though!

**Comment by agnivade** [comment]
Agree on #3.

> Maybe we could still suggest clients that are available through common package managers in the readme and leave a link to the wiki for the complete list?

I think @mebeim's suggestion above makes sense.

I would remove this line `Install it with npm install -g tldr or try the web client.` from the first section. And then have an "Install" section right after "What is tldr-pages?" where we just mention npm install, brew install etc. And just link to the wiki page for the rest.

I think #4 is fine. It makes us humble to show that this is not like a unique project that no one else has thought of.

**Comment by CleanMachine1** [comment]
This has since been done. Closing.
